frontend/src/views/AccountView.vue

```diff
@@ -46,10 +48,26 @@ export default {
     RosalutionHeader,
     SectionBox,
   },
+  data() {
+    return {
+      showSecretValue: false,
+    };
+  },
   computed: {
     user() {
       return authStore.getUser();
     },
+    secretValue() {
+      if (this.showSecretValue) {
+        return [this.user.clientSecret];
+      } else {
+        if (this.user.clientSecret) {
+          return ['<click to show>'];
+        } else {
+          return ['<empty>'];
+        }
+      }
```

Comment:
Something that can be done to clean this up, is to remove the else statement.  Since this code block would never get reached anyway when the 'if' confiditional is true due to the return statement in the true portion, it can simply before after the if statemnt and the else block removed.

```suggestion
      } 
      if (this.user.clientSecret) {
        return ['<click to show>'];
       }
       
       return ['<empty>'];
```